services/app-web/src/pages/StateSubmission/StateSubmissionForm.test.tsx

```diff
@@ -27,7 +27,7 @@ describe('StateSubmissionForm', () => {
                     apolloProvider: {
                         mocks: [
```

Comment:
I wonder if we should configure some default mocks for the apolloProvider in renderWithProviders?